e2e/browser/test/e2e.playwright.ts

```diff
@@ -20,11 +20,14 @@
 //
 
 import { test, expect } from "@playwright/test";
+import {
+  setupEnv,
+  getBrowserTestingEnvironment,
+} from "@inrupt/test-env-helpers";
 import { essUserLogin } from "./roles";
 
-import { getTestingEnvironmentBrowser } from "../../e2e-setup";
-
-const { login, password } = getTestingEnvironmentBrowser();
+setupEnv();
```

Comment:
Why doesn't `getBrowserTestingEnvironment` invoke `setupEnv()` ?

Reply:
It does and this should be a dupe, will remove.